code/game/objects/structures/orbital_cannon.dm

```diff
@@ -190,13 +193,15 @@
 			inaccurate_fuel = abs(GLOB.marine_main_ship?.ob_type_fuel_requirements[4] - tray.fuel_amt)
 
 	var/turf/target = locate(T.x + inaccurate_fuel * pick(-1,1),T.y + inaccurate_fuel * pick(-1,1),T.z)
-	for(var/i in hearers(WARHEAD_FALLING_SOUND_RANGE,target))
-		var/mob/M = i
-		M.playsound_local(target, 'sound/effects/OB_incoming.ogg', falloff = 2)
+
+	playsound_z_humans(2, 'sound/effects/OB_warning_announce.ogg', 100, FALSE, 20, 4) //for marines on ground
+	playsound(target, 'sound/effects/OB_warning_noVO.ogg', 100, FALSE, 20, 4) //for xenomorphs and anyone nearby on ground. can probably be polished to only play to xenos instead
```

Comment:
Rename this file to more accurately reflect what it plays pls

Reply:
OB_warning_announce_novoiceover.ogg sounds okay?